spec/controllers/responses_controller_spec.rb

```diff
@@ -45,5 +45,10 @@
     it 'finishes the response' do
       expect { subject }.to change { Events::ResponseFinished.count }.by 1
     end
+
+    it 'sends the email' do
+      expect(ResponseMailer).to receive(:response_email).and_call_original
```

Comment:
Klopt 😄 

Maar wil je wel `deliver_now` uitvoeren in een spec? Verder naar boven in de PR staat namelijk `allow(ResponseMailer).to receive_message_chain(:response_email, :deliver_now)`. Lijkt me in ieder geval nuttig om consistent te zijn in het wel of niet mocken van `deliver_now`